mne/cov.py

```diff
@@ -525,6 +529,10 @@ def compute_covariance(epochs, keep_sample_mean=True, tmin=None, tmax=None,
         returned (sorted by log-likelihood, from high to low, i.e. from best
         to worst).
 
+    See Also
+    --------
+    compute_raw_data_covariance : Estimate noise cov from raw data
```

Comment:
you say cov here, but just above, it is written covariance. should keep the consistency